android/app/src/main/assets/microlocations

```diff
@@ -1,106 +1,122 @@
 [
-    {
```

Comment:
Yes, but the server response has changed, we don't want to create inconsistent database state